common/pulp/common/error_codes.py

```diff
@@ -95,6 +95,11 @@
 PLP0039 = Error("PLP0039", _("The unit model with the id %(model_id)s failed to register. The "
                              "class %(model_class)s is not a subclass of ContentUnit."),
                 ['model_id', 'model_class'])
+PLP0040 = Error("PLP0040", _("Database 'seeds' config must include at least one hostname:port "
+                             "value. It can also be a comma separated list of seeds belonging to a "
```

Comment:
Consider replacing the second sentence with a reference to server.conf docs about the field's usage.